i3_agenda/API.py

```diff
@@ -67,65 +65,89 @@ def getEvents(
         page_token = calendar_list.get("nextPageToken")
         if not page_token:
             break
+    return calendar_ids
+
+
+def event_from_json(event):
+    end_time = get_event_time(event["end"].get("dateTime", event["end"].get("date")))
+    start_time = event["start"].get("dateTime", event["start"].get("date"))
+    unix_time = get_event_time(start_time)
+
+    location = None
+
+    if "location" in event:
+        location = event["location"]
+    elif "description" in event:
+        matches = re.findall(URL_REGEX, event["description"])
+        location = matches[0][0] if matches else None
+
+    return Event(
+            event["summary"],
+            is_allday(start_time),
+            unix_time,
+            end_time,
+            location,
+        )
+
+
+def get_event_result_today_only(service, calendar_id, max_results):
+    now = datetime.datetime.utcnow()
+    now_rfc3339 = now.isoformat() + "Z"  # 'Z' indicates UTC time
+    midnight_rfc3339 = (
+            now.replace(hour=23, minute=39, second=59).isoformat() + "Z"
+    )
+    return (service.events()
+            .list(
+                calendarId=calendar_id,
+                timeMin=now_rfc3339,
+                timeMax=midnight_rfc3339,
+                maxResults=max_results,
+                singleEvents=True,
+                orderBy="startTime",
+            )
+            .execute())
+
+
+def get_event_result(service, calendar_id, max_results):
```

Comment:
could return the `result.get("items", [])`, then it could just be called `get_events`.  
The result of this function is never used except for `.get(...)`, so we could just return the actual value from the `get`

Reply:
Great, its done.